Sources/OpenTelemetrySdk/Trace/Export/SpanExporter.swift

```diff
@@ -24,6 +24,9 @@ public protocol SpanExporter: AnyObject {
     /// - Parameter spans: the list of sampled Spans to be exported.
     @discardableResult func export(spans: [SpanData]) -> SpanExporterResultCode
 
+    ///Exports the collection of sampled Spans that have not yet been exported.
```

Comment:
does it just schedule for sending or actually sending it no matter how long it takes?

Reply:
This is the abstract method to be implemented, no currently exporter implements it further than simple return.